packaging/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,22 @@
+FROM registry-proxy.engineering.redhat.com/rh-osbs/rhel9-rhel_bootc:rhel-9.4
+ADD etc etc
+RUN rm -rf /opt && \
+    mkdir -p /opt/cni
+RUN dnf install -y microshift
```

Comment:
Maybe put all commands under a single `RUN` so we don't create additional layers?

Reply:
+1